ProcessMaker/Http/Controllers/Api/V1_1/TaskController.php

```diff
@@ -56,6 +57,10 @@ public function index()
 
     private function processFilters(Request $request, Builder $query)
     {
+        if (request()->has('user_id')) {
+            $query->where('user_id', request()->get('user_id'));
```

Comment:
Also, consider to use `$request` since it is declared in the function parameters.

Reply:
It was fixed